Sources/DeepLearning/Layer.swift

```diff
@@ -373,7 +482,16 @@ public struct BatchNorm<Scalar: TensorFlowFloatingPoint>: Layer {
     /// The running variance.
     @noDerivative public let runningVariance: Parameter<Scalar>
 
-    /// The batch dimension.
+    /// Creates a batch normalization layer.
+    ///
+    /// - Parameters:
+    ///   - axis: The axis that should be normalized (typically the features axis).
+    ///   - momentum: The momentum for the moving average.
+    ///   - offset: The offset to be added to the normalized tensor.
+    ///   - scale: The scale to multiply the normalized tensor by.
+    ///   - epsilon: The small scalar added to variance to avoid dividing by zero.
```

Comment:
In https://github.com/tensorflow/swift-apis/pull/46, I used the following doc comment for `epsilon`:
`A small scalar added to the denominator to improve numerical stability.`

What do you think?

Reply:
Okay. Changed.